ember/translations/form-configuration/de.yaml

```diff
@@ -2,4 +2,5 @@ form-configuration:
   title: "Konfiguration"
   expertAssociationForm: "Fachgesellschaft"
   advisoryBoardForm: "Kommission/Kuratorium"
+  internal: "Versteckt"
```

Comment:
key should be `internalForm`. Same for the other langs.

![2023-05-11_11-29-35_520x39](https://github.com/adfinis/mySAGW/assets/1833932/a0616d09-f4b2-4119-a617-195b4e80b647)

Reply:
renamed to hidden